docs/extract-static.md

```diff
@@ -4,7 +4,15 @@ title: "Extract Static"
 
 ###### [requires babel plugin](/docs/babel.md)
 
-The `extractStatic` option to `babel-plugin-emotion` allows you to extract styles with no interpolations into external css files. **`extractStatic` is not recommended** because it **breaks [composition](/docs/composition.md)** and other powerful patterns from libraries like [facepaint](https://github.com/emotion-js/facepaint). It is primarily available for historical reasons. It does not work with object styles.
+## DEPRECATED
+
+extractStatic is deprecated and will be removed in emotion@10. We recommend disabling extractStatic or using other libraries like [linaria](https://github.com/callstack-io/linaria) or [css-literal-loader](https://github.com/4Catalyzer/css-literal-loader).
+
+### Why is extractStatic being deprecated?
+
+The emotion team isn't very interested in static extraction in its current state as it negates lots of the benefits of emotion, most notably, [composition](/docs/composition.md). There is also very little interest from the community in maintaining it and maintaining `extractStatic` takes time away from maintaining more widely used parts of emotion. We don't think static extraction is a bad idea but the implementation that we have doesn't provide a good enough experience and we believe there are more important problems to be solved first. We're very interested in the work being done with [prepack](https://github.com/facebook/prepack) though and if we add some form of static extraction back, it will likely be done with prepack.
```

Comment:
I wrote this as an alternative. Let me know what you think.

> Static extraction was designed and built for earlier versions of emotion which functioned in subtly different ways than current versions. As the team added more features which expanded the power of [composition](/docs/composition.md) and made it more performant, static extraction became less important. Because most of the community is leveraging the power of composition, the time cost in maintaining `extractStatic` has become too high. Libraries such as [linaria](https://github.com/callstack-io/linaria) and [css-literal-loader](https://github.com/4Catalyzer/css-literal-loader) do static extraction well and the developers working on them are focused on this specific problem. As a team, we believe that projects like [prepack](https://github.com/facebook/prepack), when they are ready, will provide a better and more forward thinking solution to the problem or pre-compiling styles.

Reply:
I like it, I’ll probably use that with a few small changes